publish.gradle

```diff
@@ -136,13 +136,13 @@ publishing {
         }
         if(project.hasProperty("repos.external")) {
             maven {
-                name = "GitHubPackages"
-                url = uri("https://maven.pkg.github.com/northernforce/nfr-lib")
+                name = "repsy-repo"
+                url = uri("https://repo.repsy.io/mvn/cml1010101/nfr-lib")
                 credentials {
-                    username = project.findProperty("gpr.user") ?: System.getenv("GITHUB_USERNAME")
-                    password = project.findProperty("gpr.key") ?: System.getenv("GITHUB_TOKEN")
+                    username = project.findProperty("gpr.user") 
+                    password = project.findProperty("gpr.key") 
```

Comment:
```suggestion
                    password = project.findProperty("auth.key") ?: System.getenv("REPSY_PASSWORD")  
```

Reply:
First I believe gpr references GitHub packages  
Secondly, if we include the `System.getenv` It could make it easier for users making repeat  publishes

Reply:
That makes sense